api.md

```diff
@@ -0,0 +1,215 @@
+# CFPB API Standards
+
+This document captures **CFPB's view of API best practices and standards**. We aim to incorporate as many of them as possible into our work.
+
+APIs, like other web applications, vary greatly in implementation and design, depending on the situation and the problem the application is solving.
```

Comment:
Honestly this whole sentence feels superfluous. And that third comma bothers me.